pennylane/grouping/group_observables.py

```diff
@@ -47,16 +47,17 @@ class PauliGroupingStrategy:  # pylint: disable=too-many-instance-attributes
     find approximate solutions in polynomial time.
 
     Args:
-        observables (list[Observable]): A list of Pauli words to be partitioned according to a
-        grouping strategy
+        observables (list[Observable]): a list of Pauli words to be partitioned according to a
+            grouping strategy
 
     Keyword Args:
         grouping_type (str): the binary relation used to define partitions of the Pauli words
         graph_colourer (str): the heuristic algorithm to employ for graph colouring
 
     Raises:
-        ValueError: if `grouping_type` or `graph_colourer` are not recognized as elements of
-        `GROUPING_TYPES` or `GRAPH_COLOURING_METHODS` respectively
+        ValueError: if arguments specified for `grouping_type` or
+            `graph_colourer` are not recognized as elements of `GROUPING_TYPES` or
+            `GRAPH_COLOURING_METHODS` respectively
```

Comment:
Note that `GROUPING_TYPES` and `GRAPH_COLOURING_METHODS` are not displayed in the documentation, so the user reading this will not know what the elements are. Perhaps they should be duplicated in the description of `grouping_type` and `graph_colourer`.